app/Platform/Controllers/DeveloperRecentActivityController.php

```diff
@@ -0,0 +1,283 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Platform\Controllers;
+
+use App\Community\Enums\AwardType;
+use App\Http\Controller;
+use App\Platform\Enums\UnlockMode;
+use App\Platform\Models\Achievement;
+use App\Platform\Models\Game;
+use App\Platform\Models\LeaderboardEntryLegacy;
+use App\Platform\Models\PlayerAchievement;
+use App\Platform\Models\PlayerBadge;
+use App\Platform\Models\System;
+use App\Site\Models\User;
+use Illuminate\Contracts\View\View;
+use Illuminate\Http\Request;
+use Illuminate\Support\Carbon;
+
+class DeveloperRecentActivityController extends Controller
+{
+    public function __invoke(Request $request): View
+    {
+        $targetUsername = $request->route()->parameters['user'];
+        $foundTargetUser = User::where('User', 'like', $targetUsername)->first();
+        if (!$this->getCanViewTargetUser($foundTargetUser)) {
+            abort(404);
+        }
```

Comment:
This reports "not found" for any user without a ContribCount. I'd suggest widening that to any non-developer without a ContribCount. Jr. Devs who have just published their first set may want to leverage the page to find their first players.

Reply:
Widened in latest to:

```php
$targetUserPermissions = (int) $user->getAttribute('Permissions');

return (bool) $user && $targetUserPermissions >= Permissions::Registered;
```

Reply:
This throws an exception if you try to access the page for a non-existant user: http://localhost:64000/user/nonexistant/dev/feed

Reply:
I think you still want to have ContribCount factored in there - otherwise, the page exists and is always blank for players who have never done any development work, and you can't peruse the feed for banned developers.

Reply:
That makes sense. Sample implementation is much appreciated, for some reason I was having a hard time getting this concept to click. Updated in latest.